ppa/editorial/templates/editorial/snippets/attribution.html

```diff
@@ -1,10 +1,17 @@
 {# Snippet to include authors and date attribution to editorial posts #}
-{% for block in page.authors %}
-    {% if block.value.url and add_link %}
-        <a href="{{ block.value.url }}" target="_blank">
+{% if page.authors %}
```

Comment:
@bwhicks I see what you mean, there are three different cases here which make the logic more complicated. Maybe you could use `firstof` to consolidate the cases somehow, but I'm not sure. I don't love the duplication, but it's not terrible and it's certainly readable. Works for me.